<launch>
	<arg name="color" default="blue"/>
	<arg name="debug" default="true"/>
	<arg name="mode" default="1"/>
	<!-- 	mode = 0 for detection with LEDs + circle sticker
				mode = 1 for detection with only circle sticker
	-->

		<node
			pkg="base_vision"
			type="armor_detection"
			name="armor_detection"
			required="true"
			output="screen"
		>
		<param name="subscribed_image_topic" value="image_rect_color"/>
		<param name="armor_color" value="$(arg color)"/>
		<param name="mode" value="$(arg mode)"/>
		<param name="debug" value="$(arg debug)"/>
		<param name="published_topic" value="roi"/>

		</node>
</launch>
